R/as_sass.R

```diff
@@ -105,6 +105,11 @@ as_sass_.list <- function(input) {
   paste0("$", input_names, ": ", input_values, ";", collapse = "\n")
 }
 
+as_sass_.sass_layer <- function(input) {
+  sass <- as_sass_(list(input$pre, input$post))
+  attachDependencies(as_html(sass, "sass"), input$deps)
```

Comment:
Don't think `as_html` is needed here nor in `as_sass` (though they do need to be `class="sass"`)